src/Evolution/Systems/ScalarWave/Characteristics.hpp

```diff
@@ -0,0 +1,225 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <string>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Domain/FaceNormal.hpp"
+#include "Evolution/Systems/ScalarWave/Tags.hpp"
+#include "Options/Options.hpp"
+#include "Utilities/ForceInline.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+template <typename>
+class Variables;
+
+class DataVector;
+
+namespace Tags {
+template <typename Tag>
+struct Normalized;
+}  // namespace Tags
+/// \endcond
+
+namespace ScalarWave {
+namespace Tags {
+struct ConstraintGamma2Compute : ConstraintGamma2, db::ComputeTag {
```

Comment:
You need `DataStructures/DataBox/DataBoxTags.hpp` for `db::ComputeTag`

Reply:
added